components/rhs_root_post/rhs_root_post.jsx

```diff
@@ -226,19 +226,19 @@ export default class RhsRootPost extends React.PureComponent {
         });
     };
 
-    handleDotMenuOpened = (isOpened) => {
+    handleDotMenuOpened = (open) => {
         this.setState({
-            showDotMenu: isOpened,
+            showDotMenu: open,
         });
     };
 
     handleActionsMenuOpened = (open) => {
         this.setState({showActionsMenu: open});
```

Comment:
Can you unify also the style? (whether the setState is a one-liner or several lines)

Reply:
If setState is only setting one field/value, I made it a one-liner 00128f0